doc/source/release/1.15.0-notes.rst

```diff
@@ -407,6 +413,8 @@ Backwards incompatible changes
 *************
 Other changes
 *************
+- A separate accompanying type stubs package, ``scipy-stubs``, will be made
+  available with the ``1.15.0`` release.
```

Comment:
I suppose that installation instructions might help here. I've documented them at https://github.com/jorenham/scipy-stubs?tab=readme-ov-file#installation.

Now that I think about, maybe we could add it as an "extra", so that it's simply `pip scipy[stubs]`? I'm not sure what the alternative for this is for conda though, but `scipy-stubs` is available through `conda-forge` already 🤔

Reply:
it works quite nicely on conda-forge, `pixi add scipy` gives you SciPy without the tests and `pixi add scipy-tests` gives you SciPy with the tests - see https://github.com/conda-forge/scipy-feedstock/blob/1a8ff378bebd3f0b0bc6a2d1662a77d194ac59a1/recipe/meta.yaml#L205. It seems feasible to me to make something like `pixi add scipy-with-stubs` available with a runtime dependency on `scipy-stubs`.